doc/api/util.md

```diff
@@ -167,6 +167,11 @@ corresponding argument. Supported placeholders are:
 * `%f` - Floating point value.
 * `%j` - JSON.  Replaced with the string `'[Circular]'` if the argument
 contains circular references.
+* `%o` - Object. A string representation of an object 
+  with optimally useful formatting.
+* `%O` - Object. A string representation of an object 
```

Comment:
It seems we should state explicitly that JSON stringification is involved here with its restrictions.